.kokoro/presubmit/e2e.cfg

```diff
@@ -3,5 +3,10 @@
 # Only run this nox session.
 env_vars: {
     key: "NOX_SESSION"
-    value: "unit_prerelease system_prerelease system_noextras e2e notebook samples"
```

Comment:
removing only temporarily to see if the rest of the tests pass in the new project

Reply:
restored